src/main/java/com/ibm/stocator/fs/swift/SwiftOutputStream.java

```diff
@@ -133,6 +127,13 @@ public void run() {
 
   private void startThread() {
     if (writeThread.getState().equals(Thread.State.NEW)) {
+      Thread.UncaughtExceptionHandler handler = new Thread.UncaughtExceptionHandler() {
+        @Override
+        public void uncaughtException(Thread t, Throwable e) {
+          LOG.error(t.getName() + e);
```

Comment:
@djalova  the same, i think documenting exception is not enough. We need to make sure that SwiftoutputSteam will throw an exception to upper layers so operations will be stopped.